react/src/views/Transfers/ShipmentView/components/ShipmentCard.tsx

```diff
@@ -0,0 +1,113 @@
+import { Shipment } from "types/generated/graphql";
+
+export interface IShipmentProps {
+  shipment: Shipment;
+}
+
+function ShipmentCard({ shipment }: IShipmentProps) {
+  return (
+    <Box
+      boxShadow="md"
+      p="6"
+      padding={0}
+      rounded="lg"
+      bg="white"
+      width={{ base: "240pt", md: "250pt", lg: "250pt" }}
+      borderColor="blackAlpha.800"
+      borderWidth={2}
+    >
+      <VStack
+        p="6"
+        padding={0}
+        rounded="md"
+        bg="white"
+        divider={<StackDivider borderColor="blackAlpha.800" />}
+        spacing={1}
+        align="stretch"
+      >
+        <VStack spacing={2} padding={1} align="center">
+          <Box fontWeight="extrabold">{shipment?.id}</Box>
+          <Box fontWeight="xl">
+            Status:
+            {shipment?.state}
+          </Box>
+        </VStack>
+        <Box border={0}>
+          <Flex minWidth="max-content" alignItems="center" gap="2">
+            <Spacer />
+            <Box p="4">
+              <List spacing={1}>
+                <ListItem>
+                  <Flex alignContent="center">
+                    <Text fontSize="md" fontWeight="bold">
+                      {shipment?.sourceBase?.name}
+                    </Text>
+                  </Flex>
+                </ListItem>
+                <ListItem>
+                  <Flex alignContent="center">
+                    <Text fontSize="md">{shipment?.sourceBase?.organisation.name}</Text>
+                  </Flex>
+                </ListItem>
+              </List>
+            </Box>
+            <Spacer />
+            <Box>
+              <Flex alignContent="center">
+                <ShipmentIcon />
+              </Flex>
+            </Box>
+            <Spacer />
+            <Box p="4">
+              <List spacing={1}>
+                <ListItem>
+                  <Flex alignContent="center">
+                    <Text fontSize="md" fontWeight="bold">
+                      {shipment?.targetBase?.name}
+                    </Text>
+                  </Flex>
+                </ListItem>
+                <ListItem>
+                  <Flex alignContent="center">
+                    <Text fontSize="md">{shipment?.targetBase?.organisation.name}</Text>
+                  </Flex>
+                </ListItem>
+              </List>
+            </Box>
+            <Spacer />
+          </Flex>
+          {typeof shipment.transferAgreement?.comment !== "undefined" && (
+            <Center alignContent="stretch">
+              <Spacer />
+              <SpecialNoteIcon />
+              <Text fontStyle="italic" p={2}>
+                “{shipment.transferAgreement?.comment}”
+              </Text>
+              <Spacer />
+            </Center>
+          )}
+        </Box>
+        <Box p={4}>
+          <Center alignContent="stretch">
+            <Text fontWeight="bold">TOTAL:</Text> {shipment.details.length} BOX
```

Comment:
@HaGuesto and I just agreed that FE must filter for all details with `deletedOn === null`

Reply:
@pylipp I am pushing latest changes now, please check that version. :)

Reply:
:+1: also in the design it's `boxes` (lowercase and plural), and there should be space between `TOTAL:` and the count

Reply:
@pylipp I have not change it as @aerinsol working on the ui for that part